Sources/SwiftDocC/Benchmark/Benchmark.swift

```diff
@@ -130,3 +122,9 @@ public func benchmark<E, Result>(wrap event: @autoclosure () -> E, benchmarkLog
         return try body()
     }
 }
+
+private extension BenchmarkMetric {
+    static func canBenchmark(log: Benchmark) -> Bool {
```

Comment:
This reads a bit odd to me, because it could be inferred as benchmarking the log. I’m wondering if flipping the extension here would read nicer:

```swift
private extension Benchmark {
    func shouldLogMetric(_ metric: BenchmarkMetric) -> Bool {
        …
    }
}
```

What do you think?

Reply:
Other than the name, I think that's a very neat optimization to extract the logic into a separate method 👍🏼

Reply:
Makes sense, good idea!